docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,224 @@
+# The following imports are necessary for physics simulations
+import numpy as np
+from fury import window, actor, ui, utils
+import itertools
+import pybullet as p
+
+# Instantiate Pybullet client.
+client = p.connect(p.GUI)
+
+# Apply gravity to the scene.
+gravity_x = 0
+gravity_y = 0
+gravity_z = -10
+p.setGravity(gravity_x, gravity_y, gravity_z, physicsClientId=client)
+
+# Base Plane Parameters
+base_size = np.array([15, 15, 0.2])
+base_color = np.array([1, 1, 1])
+base_position = np.array([0, 0, -0.1])
+base_orientation = np.array([0, 0, 0, 1])
+
+
+# Creating BASE Plane
+base_actor = actor.box(centers=np.array([[0, 0, 0]]),
+                       directions=[0, 0, 0],
+                       scales=base_size,
+                       colors=base_color)
+
+base_coll = p.createCollisionShape(p.GEOM_BOX,
+                                   halfExtents=base_size / 2)
+
+# half of the actual size.
+base = p.createMultiBody(
+    baseCollisionShapeIndex=base_coll,
+    basePosition=base_position,
+    baseOrientation=base_orientation)
+
+p.changeDynamics(base, -1, lateralFriction=0.3, restitution=0.5)
+
+
+# Brick Parameters
+brick_mass = 0.5
+brick_size = np.array([0.8,2, 0.2])
+
+nb_bricks = 10
+brick_centers = np.zeros((nb_bricks, 3))
+
+# Keep all the Bricks Parallel
+brick_directions = np.zeros((nb_bricks, 3))
+brick_directions[:] = np.array([1, 0, 1])
+
+brick_orns = np.zeros((nb_bricks, 4))
+
+brick_sizes = np.zeros((nb_bricks, 3))
+brick_sizes[:] = brick_size
+
+brick_colors = np.random.rand(nb_bricks, 3)
+
+brick_coll = p.createCollisionShape(p.GEOM_BOX,
+                                    halfExtents=brick_size / 2)
+
+# We use this array to store the reference of brick objects in pybullet world.
+bricks = np.zeros(nb_bricks, dtype=np.int8)
+
+
+n_dominoes = 10
+centers_list = np.zeros((n_dominoes, 3))
+
+# Adding the dominoes
+for i in range(n_dominoes):
+    center_pos = np.array([(i*0.99)-5,0.4,1])
+    brick_centers[i] = center_pos
+    if i == 0 : 
+        brick_orns[0] = np.array([-0.6 , -0.6, 1, 1])
+    else:
+        brick_orns[i] = np.array([1 ,1, 1, 1])
```

Comment:
@SanjayMarreddi This is the part of the code that is ruining the positioning of the bricks. These values are in `quaternion`. Remove this part and the simulation should work fine.

Reply:
@Nibba2018 
I kept the Orientation of the First Brick in a Slanted Position willingly. So that the Brick will yield to Domino effect even though we apply Force at COM. (We can even change this scenario by keeping the Brick Parallel only and apply force above COM as you said)

**But my Doubt is the Following:-
Currently, With the code (with out removing above 73-76 lines), The Simulation is working Correctly in the PyBullet World(GUI). But in the FURY World, The Dominoes are coming in "Horizontal position" because of which the Simulation in FURY world is not completely Correct.** 

Please find the Images of the Same:-

1) In the Pybullet :
![Screenshot (191)](https://user-images.githubusercontent.com/57671048/97831526-4f209500-1cf6-11eb-9a87-610b605d4852.png)

2) In the Fury:
![Screenshot (192)](https://user-images.githubusercontent.com/57671048/97831560-695a7300-1cf6-11eb-8a94-c0ffb0a9a246.png)

Reply:
@SanjayMarreddi In these lines you are explicitly assigning values to `brick_orns` which is only used by pybullet and not by fury. If you need to make changes to the fury world then you have to update `brick_directions` as well. Now as you are only updating `brick_orns` instead of updating both, it means that your simulation in fury will have a negative offset(in orientation) compared to the one in pybullet.

Just remove those line and you will notice that pybullet and fury are both in sync. Pybullet and Fury uses different methods to represent orientations and we haven't figured out a way to map those together. So, we let `sync_bricks` handle the orientation situation during runtime.

Reply:
@Nibba2018 
I understood your explanation. But the problem is If I remove those 4 lines, I could not see any thing in both the FURY and PYBULLET WORLD !

So I am trying to change `brick_directions`

Reply:
@SanjayMarreddi right for that you need to set `brick_directions[:] = np.array([1.57, 0, 0])` and `brick_orns[i] = np.array([0, 0, 0, 1])`

Reply:
@Nibba2018  Basically I started from this only (In the first Commit of PR).
Since it is Initializing the things in a Wrong way, I started changing the Orientation to   
```
brick_orns[i] = np.array([1 ,1, 1, 1])
``` 
to make the Bricks Vertical in Pybullet (GUI).

In order to apply force on the COM on 1st Brick, I tilted the Orientation of 1st Brick and finally changed to -

```    
if i == 0 : 
        brick_orns[0] = np.array([-0.6 , -0.6, 1, 1])
    else:
        brick_orns[i] = np.array([1 ,1, 1, 1])
```
So Now it is working Fine in GUI

and then I tried Multiple Directions for FURY world to fix orientation problem in FURY like -
```
brick_directions[:] = np.array([1.57, 0, 0])
brick_directions[:] = np.array([0, 1, 1]) # This is giving better result
```

The result corresponding to `brick_directions[:] = np.array([0, 1, 1])` is as follows:-

![Screenshot (197)](https://user-images.githubusercontent.com/57671048/97846299-cca6ce00-1d13-11eb-93a7-cabe540cefe8.png)

**The only thing left is to make this similar to the Pybullet by changing `brick_directions[:] = np.array([0, 1, 1])` to a better value. I have been trying soo hard for this.. Please suggest the direction.**

Thanks!

Reply:
@SanjayMarreddi Is it really necessary to tilt the domino? I mean you are already applying a force in `timer_callback`. Trying to tilt it will complicate things. I have made a PR on your repo kindly check.

Reply:
@Nibba2018 

Okay Got it.. Thanks .!

So finally, The entire Problem between the Syncing of FURY and PYBULLET occurred because of Including the Tilt in the First domino. So to rectify this we kept it parallel only and applied force at a point above COM..